cpu/stm32/stmclk/Makefile

```diff
@@ -14,6 +16,13 @@ else ifneq (,$(filter $(CPU_FAM),l5))
   SRC += stmclk_l5.c
 else ifneq (,$(filter $(CPU_FAM),g0 g4))
   SRC += stmclk_gx.c
+else ifneq (,$(filter $(CPU_FAM),mp1))
+  ifneq (,$(filter stm32mp1_eng_mode,$(USEMODULE)))
+    SRC = stmclk_mp1.c
+  else
+	# Do not use *.c as SRC if SRC is empty (see Makefile.base)
+	NO_AUTO_SRC = 1
```

Comment:
Are you sure ? `SRC` contains `stmclk_common.c` for mp1 so it shouldn't be empty in non engineering mode.